wiki/Performance_Troubleshooting/en.md

```diff
@@ -86,3 +86,22 @@ While this may seem like the best option (and indeed can be, if your hardware ca
 - GPUs generally aren’t meant to render 1000+ frames per second, so unexpected behaviour may occur, including lower performance or higher latency than otherwise attainable.
 
 Even so, if heat generation and potentially overworking your hardware is not considered an issue, unlimited may be the option that provides you the most stable frame times.
+
+### Custom
+
+This frame limiter attempts to keep a frame rate equal to your custom preference. However, this is a setting that cannot be changed to within the in-game menu.
+
+To change osu! custom maximum framerate, you'll need to tweak some values in the config file. Here's the steps:
+
+- Open your osu! folder. (The default is `C:\Users\<yourPCusername>\AppData\Roaming\osu!`.)
+- Open the `osu!.<yourPCusername>.cfg` file with Notepad or any text editors.
+- Find the line `FrameSync = <value>` and change the value to `Custom`.
+- Find the line `CustomFrameLimit = <value>` and change the value to your preference.
+
+This is our recommended frame limiter for users who:
+
+- Want to set the frame limiter at 4x refresh rate, etc. but the game doesn't have the option to do so.
+- Doesn't satisfied with the maximum of 960fps that Optimal gives (but **do not** want to run their CPU/GPU to extremities), resulting in even lower input latency than Optimal without abusing your CPU/GPU too much with Unlimited.
+
+Note that, if you're using cuttingedge Release Stream, if you change the `CustomFrameLimit` to any number above 999, it will be reverted to 999. On Stable, you can set the value above 1000 normally.
+Also, you can not switch into this setting with F7 shortcut in-game, even if you had done the steps above, (only available via the Settings menu). And you'll need to switch to Custom in the Settings before quitting the game, so it'll be available the next time osu! runs. Else, you'll have to change the `FrameSync` manually again.
```

Comment:
```suggestion
Note that, if you're using the Cutting Edge release stream, changing the `CustomFrameLimit` to any number above 999, will be reverted to 999; on Stable, you can set the value above 1000 with no issues.

Also note that you cannot switch into this setting with the `F7` shortcut in-game, even if you had done the steps above: you'll need to switch to Custom in the Settings before quitting the game so it'll be available the next time osu! runs. Otherwise, you'll have to change the `FrameSync` manually again.
```
Remark; also these should probably be hatnotes

Reply:
I've changed to hatnotes!